src/snowflake/snowpark/mock/mock_plan.py

```diff
@@ -528,18 +537,27 @@ def calculate_expression(
     input_data: Union[TableEmulator, ColumnEmulator],
     analyzer,
     expr_to_alias: Dict[str, str],
-) -> ColumnEmulator:
+    *,
+    keep_literal: bool = False,
+) -> Union[ColumnEmulator, object]:
```

Comment:
Does the `object` here correspond to when `keep_literal` is True?

Reply:
comment updated